nixos/modules/services/networking/twingate.nix

```diff
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.twingate;
+
+in {
+
+  options.services.twingate = {
+    enable = mkEnableOption (lib.mdDoc "Twingate Client daemon");
+  };
+
+  config = mkIf cfg.enable {
+
+    networking.firewall.checkReversePath = lib.mkDefault false;
+    networking.networkmanager.enable = true;
```

Comment:
This needs a proper assertion. You can enable networkd and networkmanager but if done incorrect it can break your networking setup and I wouldn't expect that twingate turns on networkmanager.

Reply:
This PR has been running for some time now. I'm going to go ahead and merge it and let @antonshkurenko deal with this later.